server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java

```diff
@@ -1381,6 +1382,11 @@ private void loadTimedService(Properties properties) {
     if (unseqMemTableFlushCheckInterval > 0) {
       conf.setUnseqMemtableFlushCheckInterval(unseqMemTableFlushCheckInterval);
     }
+
+    conf.setTvListSortAlgorithm(
+        TVListSortAlgorithm.valueOf(
+            properties.getProperty(
+                "tvList_sort_algorithm", conf.getTvListSortAlgorithm().toString())));
```

Comment:
Why added this in `loadTimedService` method?

Reply:
fixed, move it to the function loadProperties in line 395.